static/src/js/util/accessMethods/defaultConcatenation.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Determine whether or not the provided UMM S record supports shapefile subsetting
+ * @param {Object} service UMM S record to parse
+ */
+export const defaultConcatenation = (service) => {
+  const { serviceOptions = {} } = service
+
+  // If there are no service options the record can not support variable subsetting
```

Comment:
"the record can not support variable subsetting" Was this copied from another method? I don't see any variable subsetting in this method